packages/js/components/src/table/table.js

```diff
@@ -148,10 +149,15 @@ class Table extends Component {
 			rows,
 		} = this.props;
 		const { isScrollableRight, isScrollableLeft, tabIndex } = this.state;
-		const classes = classnames( 'woocommerce-table__table', classNames, {
-			'is-scrollable-right': isScrollableRight,
-			'is-scrollable-left': isScrollableLeft,
-		} );
+		const classes = classnames(
+			'woocommerce-table__table',
+			classNames,
```

Comment:
Good catch! 😄  @ecgan  

I think we can log a message for `classNames` using [@wordpress/deprecated](https://www.npmjs.com/package/@wordpress/deprecated). What do you think?

Reply:
I was looking for a way to mark the `classNames` prop as deprecated (was looking for something like `@deprecated`), and TIL about [@wordpress/deprecated](https://www.npmjs.com/package/@wordpress/deprecated). 😄 

I made the change in [`42ca68e` (#36151)](https://github.com/woocommerce/woocommerce/pull/36151/commits/42ca68e1a53c217d76ac733cfbb9dddb173f7c6c). I use `since: '11.1.0'` (current version) and `version: '12.0.0'` (next major version), does that make sense? Do you know what is our process or strategy to remove the deprecated prop? 🤔 

With the code change, I can see this warning message in the browser console: "Table component's classNames prop is deprecated since version 11.1.0 and will be removed from @woocommerce/components in version 12.0.0. Please use className instead."

Reply:
> I made the change in [42ca68e (#36151)](https://github.com/woocommerce/woocommerce/pull/36151/commits/42ca68e1a53c217d76ac733cfbb9dddb173f7c6c). I use since: '11.1.0' (current version) and version: '12.0.0' (next major version), does that make sense? Do you know what is our process or strategy to remove the deprecated prop? 🤔

Yes, that makes sense. You can add an issue in our backlog to remove this deprecated prop in the future. 🙌 

> With the code change, I can see this warning message in the browser console: "Table component's classNames prop is deprecated since version 11.1.0 and will be removed from @woocommerce/components in version 12.0.0. Please use className instead."

Thanks! Looks good to me. 🙂

Reply:
I have created issue https://github.com/woocommerce/woocommerce/issues/36184 to remove the deprecated prop in the future.